src/style.css

```diff
@@ -1 +1,39 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
```

Comment:
Be consistent with your margins. Since you're adding a right margin to the `.stars__star` elements, ensure you're not adding both top and bottom margins elsewhere to avoid potential margin collapse issues.

Reply:
?????